pkgs/applications/editors/vim/configurable.nix

```diff
@@ -92,26 +95,9 @@ composableDerivation {
 
       // edf {
         name = "python";
-        feat = "pythoninterp";
+        feat = "python${pythonVersion}interp";
         enable = {
-          nativeBuildInputs = [ python ];
-        } // lib.optionalAttrs stdenv.isDarwin {
```

Comment:
It's not needed, at least in my experience. It automatically checks for the location, though I actually didn't test with python2...